pkg/report/linux.go

```diff
@@ -536,67 +558,67 @@ var linuxOopses = []*oops{
 		[]oopsFormat{
 			{
 				compile("WARNING: .* at {{SRC}} {{FUNC}}"),
-				nil,
+				compile("WARNING: (?:.*\\n)+?.*Call Trace:"),
 				"WARNING in %[2]v",
 				false,
 			},
 			{
 				compile("WARNING: possible circular locking dependency detected(?:.*\\n)+?.*is trying to acquire lock(?:.*\\n)+?.*at: {{PC}} +{{FUNC}}"),
-				nil,
+				compile("WARNING: (?:.*\\n)+?.*Call Trace:"),
 				"possible deadlock in %[1]v",
 				false,
 			},
 			{
 				compile("WARNING: possible circular locking dependency detected"),
-				nil,
+				compile("WARNING: (?:.*\\n)+?.*Call Trace:"),
 				"possible deadlock",
 				false,
 			},
 			{
 				compile("WARNING: possible irq lock inversion dependency detected(?:.*\\n)+?.*just changed the state of lock(?:.*\\n)+?.*at: {{PC}} +{{FUNC}}"),
-				nil,
+				compile("WARNING: (?:.*\\n)+?.*Call Trace:"),
 				"possible deadlock in %[1]v",
 				false,
 			},
 			{
 				compile("WARNING: possible irq lock inversion dependency detected"),
-				nil,
+				compile("WARNING: (?:.*\\n)+?.*Call Trace:"),
 				"possible deadlock",
 				false,
 			},
 			{
 				compile("WARNING: SOFTIRQ-safe -> SOFTIRQ-unsafe lock order detected(?:.*\\n)+?.*is trying to acquire(?:.*\\n)+?.*at: {{PC}} +{{FUNC}}"),
-				nil,
+				compile("WARNING: (?:.*\\n)+?.*Call Trace:"),
 				"possible deadlock in %[1]v",
 				false,
 			},
 			{
 				compile("WARNING: SOFTIRQ-safe -> SOFTIRQ-unsafe lock order detected"),
-				nil,
+				compile("WARNING: (?:.*\\n)+?.*Call Trace:"),
 				"possible deadlock",
 				false,
 			},
 			{
 				compile("WARNING: possible recursive locking detected(?:.*\\n)+?.*is trying to acquire lock(?:.*\\n)+?.*at: {{PC}} +{{FUNC}}"),
-				nil,
+				compile("WARNING: (?:.*\\n)+?.*Call Trace:"),
```

Comment:
This regexp is repeated too many times.
It seems we need to add `NoStackTrace bool` field. And start naming fields. Then we can add this flag to just few formats that don't have stack trace:
```
    {
       Title: compile(...),
       NoStackTrace: true,
    },
```

Reply:
Done.